<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">.lst-kix_48yrrib0wnrf-6>li{counter-increment:lst-ctn-kix_48yrrib0wnrf-6}ol.lst-kix_tpm0vcvzdlxe-5.start{counter-reset:lst-ctn-kix_tpm0vcvzdlxe-5 0}ol.lst-kix_bxuzqw2beezn-7.start{counter-reset:lst-ctn-kix_bxuzqw2beezn-7 0}.lst-kix_bxuzqw2beezn-2>li{counter-increment:lst-ctn-kix_bxuzqw2beezn-2}.lst-kix_vgisgt4vep2-8>li:before{content:"" counter(lst-ctn-kix_vgisgt4vep2-8,lower-roman) ". "}.lst-kix_vgisgt4vep2-2>li{counter-increment:lst-ctn-kix_vgisgt4vep2-2}.lst-kix_vgisgt4vep2-7>li:before{content:"" counter(lst-ctn-kix_vgisgt4vep2-7,lower-latin) ". "}.lst-kix_tpm0vcvzdlxe-5>li{counter-increment:lst-ctn-kix_tpm0vcvzdlxe-5}ol.lst-kix_48yrrib0wnrf-0.start{counter-reset:lst-ctn-kix_48yrrib0wnrf-0 0}.lst-kix_48yrrib0wnrf-5>li{counter-increment:lst-ctn-kix_48yrrib0wnrf-5}ol.lst-kix_48yrrib0wnrf-8.start{counter-reset:lst-ctn-kix_48yrrib0wnrf-8 0}.lst-kix_vgisgt4vep2-1>li{counter-increment:lst-ctn-kix_vgisgt4vep2-1}ol.lst-kix_tpm0vcvzdlxe-8.start{counter-reset:lst-ctn-kix_tpm0vcvzdlxe-8 0}.lst-kix_bxuzqw2beezn-3>li{counter-increment:lst-ctn-kix_bxuzqw2beezn-3}.lst-kix_vgisgt4vep2-0>li{counter-increment:lst-ctn-kix_vgisgt4vep2-0}.lst-kix_bxuzqw2beezn-0>li{counter-increment:lst-ctn-kix_bxuzqw2beezn-0}ol.lst-kix_48yrrib0wnrf-3.start{counter-reset:lst-ctn-kix_48yrrib0wnrf-3 0}.lst-kix_tpm0vcvzdlxe-7>li{counter-increment:lst-ctn-kix_tpm0vcvzdlxe-7}ol.lst-kix_bxuzqw2beezn-2.start{counter-reset:lst-ctn-kix_bxuzqw2beezn-2 0}ol.lst-kix_tpm0vcvzdlxe-2.start{counter-reset:lst-ctn-kix_tpm0vcvzdlxe-2 0}.lst-kix_vgisgt4vep2-3>li{counter-increment:lst-ctn-kix_vgisgt4vep2-3}ol.lst-kix_vgisgt4vep2-7.start{counter-reset:lst-ctn-kix_vgisgt4vep2-7 0}ol.lst-kix_vgisgt4vep2-3{list-style-type:none}.lst-kix_48yrrib0wnrf-3>li{counter-increment:lst-ctn-kix_48yrrib0wnrf-3}ol.lst-kix_vgisgt4vep2-2{list-style-type:none}ol.lst-kix_vgisgt4vep2-5{list-style-type:none}ol.lst-kix_vgisgt4vep2-4{list-style-type:none}ol.lst-kix_vgisgt4vep2-1{list-style-type:none}.lst-kix_tpm0vcvzdlxe-3>li{counter-increment:lst-ctn-kix_tpm0vcvzdlxe-3}ol.lst-kix_vgisgt4vep2-0{list-style-type:none}ol.lst-kix_vgisgt4vep2-4.start{counter-reset:lst-ctn-kix_vgisgt4vep2-4 0}.lst-kix_tpm0vcvzdlxe-0>li:before{content:"" counter(lst-ctn-kix_tpm0vcvzdlxe-0,decimal) ") "}.lst-kix_tpm0vcvzdlxe-1>li:before{content:"" counter(lst-ctn-kix_tpm0vcvzdlxe-1,lower-latin) ") "}ul.lst-kix_upl8f2hos2u7-1{list-style-type:none}.lst-kix_vgisgt4vep2-5>li{counter-increment:lst-ctn-kix_vgisgt4vep2-5}ul.lst-kix_upl8f2hos2u7-0{list-style-type:none}ol.lst-kix_vgisgt4vep2-0.start{counter-reset:lst-ctn-kix_vgisgt4vep2-0 0}ol.lst-kix_tpm0vcvzdlxe-3.start{counter-reset:lst-ctn-kix_tpm0vcvzdlxe-3 0}.lst-kix_48yrrib0wnrf-8>li{counter-increment:lst-ctn-kix_48yrrib0wnrf-8}ul.lst-kix_upl8f2hos2u7-5{list-style-type:none}ul.lst-kix_upl8f2hos2u7-4{list-style-type:none}ul.lst-kix_upl8f2hos2u7-3{list-style-type:none}ul.lst-kix_upl8f2hos2u7-2{list-style-type:none}ol.lst-kix_48yrrib0wnrf-5.start{counter-reset:lst-ctn-kix_48yrrib0wnrf-5 0}ol.lst-kix_bxuzqw2beezn-2{list-style-type:none}.lst-kix_48yrrib0wnrf-2>li{counter-increment:lst-ctn-kix_48yrrib0wnrf-2}ol.lst-kix_bxuzqw2beezn-3{list-style-type:none}ol.lst-kix_bxuzqw2beezn-4{list-style-type:none}ol.lst-kix_bxuzqw2beezn-5{list-style-type:none}ol.lst-kix_bxuzqw2beezn-6{list-style-type:none}ol.lst-kix_vgisgt4vep2-7{list-style-type:none}ol.lst-kix_bxuzqw2beezn-7{list-style-type:none}ol.lst-kix_vgisgt4vep2-6{list-style-type:none}ol.lst-kix_bxuzqw2beezn-8{list-style-type:none}ol.lst-kix_bxuzqw2beezn-8.start{counter-reset:lst-ctn-kix_bxuzqw2beezn-8 0}ol.lst-kix_vgisgt4vep2-8{list-style-type:none}ol.lst-kix_vgisgt4vep2-5.start{counter-reset:lst-ctn-kix_vgisgt4vep2-5 0}ul.lst-kix_upl8f2hos2u7-8{list-style-type:none}ol.lst-kix_bxuzqw2beezn-1.start{counter-reset:lst-ctn-kix_bxuzqw2beezn-1 0}ul.lst-kix_upl8f2hos2u7-7{list-style-type:none}ul.lst-kix_upl8f2hos2u7-6{list-style-type:none}.lst-kix_bxuzqw2beezn-1>li{counter-increment:lst-ctn-kix_bxuzqw2beezn-1}.lst-kix_bxuzqw2beezn-4>li{counter-increment:lst-ctn-kix_bxuzqw2beezn-4}.lst-kix_tpm0vcvzdlxe-6>li{counter-increment:lst-ctn-kix_tpm0vcvzdlxe-6}ol.lst-kix_tpm0vcvzdlxe-4.start{counter-reset:lst-ctn-kix_tpm0vcvzdlxe-4 0}ol.lst-kix_48yrrib0wnrf-4.start{counter-reset:lst-ctn-kix_48yrrib0wnrf-4 0}ol.lst-kix_bxuzqw2beezn-0.start{counter-reset:lst-ctn-kix_bxuzqw2beezn-0 0}.lst-kix_48yrrib0wnrf-0>li{counter-increment:lst-ctn-kix_48yrrib0wnrf-0}.lst-kix_tpm0vcvzdlxe-0>li{counter-increment:lst-ctn-kix_tpm0vcvzdlxe-0}.lst-kix_48yrrib0wnrf-3>li:before{content:"" counter(lst-ctn-kix_48yrrib0wnrf-3,decimal) ". "}.lst-kix_bxuzqw2beezn-7>li{counter-increment:lst-ctn-kix_bxuzqw2beezn-7}.lst-kix_vgisgt4vep2-7>li{counter-increment:lst-ctn-kix_vgisgt4vep2-7}.lst-kix_48yrrib0wnrf-4>li:before{content:"" counter(lst-ctn-kix_48yrrib0wnrf-4,lower-latin) ". "}.lst-kix_vgisgt4vep2-8>li{counter-increment:lst-ctn-kix_vgisgt4vep2-8}ol.lst-kix_bxuzqw2beezn-3.start{counter-reset:lst-ctn-kix_bxuzqw2beezn-3 0}.lst-kix_48yrrib0wnrf-6>li:before{content:"" counter(lst-ctn-kix_48yrrib0wnrf-6,decimal) ". "}ol.lst-kix_vgisgt4vep2-6.start{counter-reset:lst-ctn-kix_vgisgt4vep2-6 0}.lst-kix_48yrrib0wnrf-5>li:before{content:"" counter(lst-ctn-kix_48yrrib0wnrf-5,lower-roman) ". "}.lst-kix_48yrrib0wnrf-8>li:before{content:"" counter(lst-ctn-kix_48yrrib0wnrf-8,lower-roman) ". "}.lst-kix_48yrrib0wnrf-7>li:before{content:"" counter(lst-ctn-kix_48yrrib0wnrf-7,lower-latin) ". "}ol.lst-kix_vgisgt4vep2-3.start{counter-reset:lst-ctn-kix_vgisgt4vep2-3 0}.lst-kix_tpm0vcvzdlxe-1>li{counter-increment:lst-ctn-kix_tpm0vcvzdlxe-1}ol.lst-kix_bxuzqw2beezn-6.start{counter-reset:lst-ctn-kix_bxuzqw2beezn-6 0}ol.lst-kix_tpm0vcvzdlxe-6.start{counter-reset:lst-ctn-kix_tpm0vcvzdlxe-6 0}.lst-kix_upl8f2hos2u7-4>li:before{content:"-  "}.lst-kix_upl8f2hos2u7-6>li:before{content:"-  "}ol.lst-kix_bxuzqw2beezn-0{list-style-type:none}ol.lst-kix_bxuzqw2beezn-1{list-style-type:none}.lst-kix_upl8f2hos2u7-3>li:before{content:"-  "}.lst-kix_upl8f2hos2u7-7>li:before{content:"-  "}.lst-kix_tpm0vcvzdlxe-8>li:before{content:"" counter(lst-ctn-kix_tpm0vcvzdlxe-8,lower-roman) ". "}.lst-kix_upl8f2hos2u7-0>li:before{content:"-  "}.lst-kix_upl8f2hos2u7-2>li:before{content:"-  "}.lst-kix_upl8f2hos2u7-8>li:before{content:"-  "}ol.lst-kix_vgisgt4vep2-1.start{counter-reset:lst-ctn-kix_vgisgt4vep2-1 0}.lst-kix_tpm0vcvzdlxe-7>li:before{content:"" counter(lst-ctn-kix_tpm0vcvzdlxe-7,lower-latin) ". "}.lst-kix_upl8f2hos2u7-1>li:before{content:"-  "}.lst-kix_tpm0vcvzdlxe-2>li{counter-increment:lst-ctn-kix_tpm0vcvzdlxe-2}.lst-kix_tpm0vcvzdlxe-4>li:before{content:"(" counter(lst-ctn-kix_tpm0vcvzdlxe-4,lower-latin) ") "}.lst-kix_tpm0vcvzdlxe-5>li:before{content:"(" counter(lst-ctn-kix_tpm0vcvzdlxe-5,lower-roman) ") "}.lst-kix_bxuzqw2beezn-8>li{counter-increment:lst-ctn-kix_bxuzqw2beezn-8}.lst-kix_48yrrib0wnrf-2>li:before{content:"" counter(lst-ctn-kix_48yrrib0wnrf-2,lower-roman) ". "}.lst-kix_bxuzqw2beezn-5>li{counter-increment:lst-ctn-kix_bxuzqw2beezn-5}.lst-kix_tpm0vcvzdlxe-2>li:before{content:"" counter(lst-ctn-kix_tpm0vcvzdlxe-2,lower-roman) ") "}.lst-kix_tpm0vcvzdlxe-6>li:before{content:"" counter(lst-ctn-kix_tpm0vcvzdlxe-6,decimal) ". "}.lst-kix_48yrrib0wnrf-0>li:before{content:"" counter(lst-ctn-kix_48yrrib0wnrf-0,decimal) ". "}.lst-kix_48yrrib0wnrf-1>li:before{content:"" counter(lst-ctn-kix_48yrrib0wnrf-1,lower-latin) ". "}.lst-kix_vgisgt4vep2-6>li{counter-increment:lst-ctn-kix_vgisgt4vep2-6}.lst-kix_tpm0vcvzdlxe-3>li:before{content:"(" counter(lst-ctn-kix_tpm0vcvzdlxe-3,decimal) ") "}.lst-kix_upl8f2hos2u7-5>li:before{content:"-  "}ol.lst-kix_bxuzqw2beezn-5.start{counter-reset:lst-ctn-kix_bxuzqw2beezn-5 0}ol.lst-kix_tpm0vcvzdlxe-0{list-style-type:none}ol.lst-kix_tpm0vcvzdlxe-0.start{counter-reset:lst-ctn-kix_tpm0vcvzdlxe-0 0}ol.lst-kix_tpm0vcvzdlxe-7.start{counter-reset:lst-ctn-kix_tpm0vcvzdlxe-7 0}.lst-kix_bxuzqw2beezn-8>li:before{content:"" counter(lst-ctn-kix_bxuzqw2beezn-8,lower-roman) ". "}ol.lst-kix_tpm0vcvzdlxe-1{list-style-type:none}.lst-kix_bxuzqw2beezn-6>li:before{content:"" counter(lst-ctn-kix_bxuzqw2beezn-6,decimal) ". "}ol.lst-kix_48yrrib0wnrf-2.start{counter-reset:lst-ctn-kix_48yrrib0wnrf-2 0}.lst-kix_vgisgt4vep2-4>li{counter-increment:lst-ctn-kix_vgisgt4vep2-4}.lst-kix_bxuzqw2beezn-5>li:before{content:"" counter(lst-ctn-kix_bxuzqw2beezn-5,lower-roman) ". "}.lst-kix_bxuzqw2beezn-7>li:before{content:"" counter(lst-ctn-kix_bxuzqw2beezn-7,lower-latin) ". "}ol.lst-kix_48yrrib0wnrf-6.start{counter-reset:lst-ctn-kix_48yrrib0wnrf-6 0}ol.lst-kix_48yrrib0wnrf-6{list-style-type:none}.lst-kix_bxuzqw2beezn-2>li:before{content:"" counter(lst-ctn-kix_bxuzqw2beezn-2,lower-roman) ". "}ol.lst-kix_48yrrib0wnrf-7{list-style-type:none}ol.lst-kix_48yrrib0wnrf-8{list-style-type:none}.lst-kix_bxuzqw2beezn-1>li:before{content:"" counter(lst-ctn-kix_bxuzqw2beezn-1,lower-latin) ". "}.lst-kix_bxuzqw2beezn-3>li:before{content:"" counter(lst-ctn-kix_bxuzqw2beezn-3,decimal) ". "}.lst-kix_tpm0vcvzdlxe-8>li{counter-increment:lst-ctn-kix_tpm0vcvzdlxe-8}ol.lst-kix_48yrrib0wnrf-2{list-style-type:none}.lst-kix_bxuzqw2beezn-0>li:before{content:"" counter(lst-ctn-kix_bxuzqw2beezn-0,decimal) ". "}.lst-kix_bxuzqw2beezn-4>li:before{content:"" counter(lst-ctn-kix_bxuzqw2beezn-4,lower-latin) ". "}ol.lst-kix_48yrrib0wnrf-3{list-style-type:none}ol.lst-kix_48yrrib0wnrf-4{list-style-type:none}ol.lst-kix_48yrrib0wnrf-5{list-style-type:none}ol.lst-kix_vgisgt4vep2-8.start{counter-reset:lst-ctn-kix_vgisgt4vep2-8 0}ol.lst-kix_48yrrib0wnrf-0{list-style-type:none}ol.lst-kix_48yrrib0wnrf-1{list-style-type:none}.lst-kix_tpm0vcvzdlxe-4>li{counter-increment:lst-ctn-kix_tpm0vcvzdlxe-4}.lst-kix_bxuzqw2beezn-6>li{counter-increment:lst-ctn-kix_bxuzqw2beezn-6}.lst-kix_vgisgt4vep2-1>li:before{content:"" counter(lst-ctn-kix_vgisgt4vep2-1,lower-latin) ". "}ol.lst-kix_tpm0vcvzdlxe-1.start{counter-reset:lst-ctn-kix_tpm0vcvzdlxe-1 0}.lst-kix_vgisgt4vep2-0>li:before{content:"" counter(lst-ctn-kix_vgisgt4vep2-0,decimal) ". "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_vgisgt4vep2-6>li:before{content:"" counter(lst-ctn-kix_vgisgt4vep2-6,decimal) ". "}.lst-kix_48yrrib0wnrf-1>li{counter-increment:lst-ctn-kix_48yrrib0wnrf-1}ol.lst-kix_48yrrib0wnrf-1.start{counter-reset:lst-ctn-kix_48yrrib0wnrf-1 0}.lst-kix_48yrrib0wnrf-4>li{counter-increment:lst-ctn-kix_48yrrib0wnrf-4}ol.lst-kix_48yrrib0wnrf-7.start{counter-reset:lst-ctn-kix_48yrrib0wnrf-7 0}.lst-kix_vgisgt4vep2-5>li:before{content:"" counter(lst-ctn-kix_vgisgt4vep2-5,lower-roman) ". "}.lst-kix_vgisgt4vep2-2>li:before{content:"" counter(lst-ctn-kix_vgisgt4vep2-2,lower-roman) ". "}ol.lst-kix_tpm0vcvzdlxe-2{list-style-type:none}.lst-kix_48yrrib0wnrf-7>li{counter-increment:lst-ctn-kix_48yrrib0wnrf-7}ol.lst-kix_vgisgt4vep2-2.start{counter-reset:lst-ctn-kix_vgisgt4vep2-2 0}ol.lst-kix_tpm0vcvzdlxe-3{list-style-type:none}ol.lst-kix_bxuzqw2beezn-4.start{counter-reset:lst-ctn-kix_bxuzqw2beezn-4 0}ol.lst-kix_tpm0vcvzdlxe-4{list-style-type:none}ol.lst-kix_tpm0vcvzdlxe-5{list-style-type:none}.lst-kix_vgisgt4vep2-4>li:before{content:"" counter(lst-ctn-kix_vgisgt4vep2-4,lower-latin) ". "}ol.lst-kix_tpm0vcvzdlxe-6{list-style-type:none}ol.lst-kix_tpm0vcvzdlxe-7{list-style-type:none}.lst-kix_vgisgt4vep2-3>li:before{content:"" counter(lst-ctn-kix_vgisgt4vep2-3,decimal) ". "}ol.lst-kix_tpm0vcvzdlxe-8{list-style-type:none}ol{margin:0;padding:0}table td,table th{padding:0}.c17{padding-top:18pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c1{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:14pt;font-family:"Arial";font-style:normal}.c6{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:13pt;font-family:"Arial";font-style:normal}.c5{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c4{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:16pt;font-family:"Arial";font-style:normal}.c7{padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c13{color:#434343;font-weight:400;text-decoration:none;vertical-align:baseline;font-family:"Arial";font-style:normal}.c0{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c16{padding-top:18pt;padding-bottom:4pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c26{padding-top:24pt;padding-bottom:6pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c14{color:#434343;font-weight:700;text-decoration:none;vertical-align:baseline;font-family:"Arial";font-style:normal}.c29{padding-top:16pt;padding-bottom:4pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c19{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-family:"Arial";font-style:normal}.c8{text-decoration-skip-ink:none;font-size:14pt;-webkit-text-decoration-skip:none;color:#1155cc;text-decoration:underline}.c15{text-decoration-skip-ink:none;-webkit-text-decoration-skip:none;color:#1155cc;text-decoration:underline}.c18{font-weight:400;vertical-align:baseline;font-family:"Arial";font-style:normal}.c23{color:#000000;text-decoration:none;font-size:13pt}.c22{color:#000000;text-decoration:none;font-size:16pt}.c31{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c12{font-size:18pt;font-weight:700}.c11{margin-left:36pt;padding-left:0pt}.c3{color:inherit;text-decoration:inherit}.c24{padding:0;margin:0}.c28{font-size:26pt}.c27{font-size:20pt}.c10{text-indent:36pt}.c2{height:11pt}.c30{margin-left:36pt}.c9{font-size:14pt}.c25{font-size:11pt}.c21{page-break-after:avoid}.c20{font-weight:700}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c31 doc-content"><h1 class="c26" id="h.4lzoli4x365m"><span class="c19 c28">Bio322</span></h1><h2 class="c16" id="h.lb8tj1wavq2q"><span class="c19 c27">Getting started</span></h2><p class="c7"><span class="c1">In CRISPR/Cas9 experiments, we will perform</span></p><p class="c7"><span class="c1">1) Guide RNA design 2) Repair prediction 3) Outcome analysis</span></p><p class="c7"><span class="c1">There are lots of different tools that exist, here we are going to use the following web-based tools to achieve the goal.</span></p><p class="c7"><span class="c9">1) Guide RNA design -</span><span class="c9"><a class="c3" href="https://www.google.com/url?q=https://chopchop.cbu.uib.no/&amp;sa=D&amp;source=editors&amp;ust=1698959533873906&amp;usg=AOvVaw1vycuNysGYbLBTL3t8zNT7">&nbsp;</a></span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://chopchop.cbu.uib.no/&amp;sa=D&amp;source=editors&amp;ust=1698959533874240&amp;usg=AOvVaw0LpIM1eF89wJNx8dP8OJgq">CHOPCHOP</a></span></p><p class="c7"><span class="c9">2) Repair prediction -</span><span class="c9"><a class="c3" href="https://www.google.com/url?q=https://partslab.sanger.ac.uk/FORECasT&amp;sa=D&amp;source=editors&amp;ust=1698959533874524&amp;usg=AOvVaw32K1VZVDsAuVyPIELfXew9">&nbsp;</a></span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://partslab.sanger.ac.uk/FORECasT&amp;sa=D&amp;source=editors&amp;ust=1698959533874757&amp;usg=AOvVaw2IBXAYmk94vqTmhnjX_4Ha">FORECast</a></span><span class="c9">&nbsp;-</span><span class="c9"><a class="c3" href="https://www.google.com/url?q=https://indelphi.giffordlab.mit.edu/guide&amp;sa=D&amp;source=editors&amp;ust=1698959533874997&amp;usg=AOvVaw052vbC1GbSZDzOKvqEjHhP">&nbsp;</a></span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://indelphi.giffordlab.mit.edu/guide&amp;sa=D&amp;source=editors&amp;ust=1698959533875219&amp;usg=AOvVaw2e2tF9bB2M_JCO52Q9cgj_">indelphi</a></span></p><p class="c7"><span class="c1">3) Outcome analysis</span></p><p class="c0"><span class="c9">* Sanger sequencing: &nbsp;- </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://tide.nki.nl/&amp;sa=D&amp;source=editors&amp;ust=1698959533875627&amp;usg=AOvVaw2rmCtgXEfH4j6bl2Ir0rPZ">TIDE</a></span><span class="c9">&nbsp;- </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://ice.synthego.com/%23/&amp;sa=D&amp;source=editors&amp;ust=1698959533875936&amp;usg=AOvVaw2nf0MHkvQmLuYpgq3akuSe">ICE analysis</a></span></p><p class="c0"><span class="c9">* NGS data: - </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=http://crispresso2.pinellolab.org/submission&amp;sa=D&amp;source=editors&amp;ust=1698959533876294&amp;usg=AOvVaw2vsx4RRG9Zp0YtxAs3zjit">CRISPResso2</a></span><span class="c9">&nbsp;- </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=http://www.rgenome.net/cas-analyzer/%23!&amp;sa=D&amp;source=editors&amp;ust=1698959533876561&amp;usg=AOvVaw1UbUydVrao2w_9YnqzImOn">RGENE Tools</a></span><hr></p><h2 class="c17" id="h.o79utnhmi1pi"><span class="c4">sgRNA design </span></h2><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 304.00px;"><img alt="" src="images/image6.png" style="width: 624.00px; height: 304.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c7"><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://chopchop.cbu.uib.no/&amp;sa=D&amp;source=editors&amp;ust=1698959533877138&amp;usg=AOvVaw0FrKoWMrEbUx6zgnZg6GzA">CHOPCHOP</a></span></p><p class="c7"><span class="c6">&gt;Explanation</span></p><p class="c7"><span class="c18 c23">[Target]</span></p><p class="c0"><span class="c1">You can choose two options for target</span></p><p class="c0"><span class="c1">1) pasta sequence &gt; Click Paste Target<br>2) gene name from ENSEMBL gene ID, NCBI RefSeq number</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">[ In ]</span></p><p class="c0"><span class="c9">For selecting your interested species</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">[Using]</span></p><p class="c0"><span class="c1">In here we will choose Using CRISPR/Cas9, however, you can also select different Cas series or even TALEN in option. </span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">[For]</span></p><p class="c0"><span class="c1">Purpose of the experiments, Knock-out (KO), Knock-in (KI) &hellip; </span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">[Options]</span></p><p class="c0"><span class="c1">Specify the target region, Can design the Primers for the downstream analysis</span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span class="c1">Example sequence - species: Human</span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0 c30"><span class="c9">AATGTCCCCCAATGGGAAGTTCATCTGGCACTGCCCACAGGTGAGGAGGTCATGATCCCCTTCTGGAGCTCCCAACGGGCCGTGGTCTGGTTCATCATCTGTAAGAATGGCTTCAAGAGGCTCGGCTGTGGTT</span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span class="c1">Example Ensemble ID - species: Salmo salar (Ssal_v3.1)</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;slc45a2</span></p><p class="c0"><span class="c9 c20">&gt;Results</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 278.67px;"><img alt="" src="images/image15.png" style="width: 624.00px; height: 278.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span class="c1">You can see the lots of sgRNA list in the results page, and also other information like strand, GC contents, self-complementarity, mismatch (MM), Efficiency <br></span></p><p class="c0"><span class="c1">Combining the information, the algorithm suggests the rank of the sgRNA as well. </span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 305.33px;"><img alt="" src="images/image16.png" style="width: 624.00px; height: 305.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">If you click one of the sgRNA, you can see the different primers to amplify the vicinity of the target region. </span></p><p class="c0 c2"><span class="c1"></span></p><hr><p class="c0 c2"><span class="c1"></span></p><h2 class="c17" id="h.1fpqojz83xzi"><span class="c4">Repair prediction</span></h2><h3 class="c7 c21" id="h.5oimyey72wnm"><span class="c15"><a class="c3" href="https://www.google.com/url?q=https://partslab.sanger.ac.uk/FORECasT&amp;sa=D&amp;source=editors&amp;ust=1698959533880578&amp;usg=AOvVaw2yviM8Y5tkVredDgIBMPpp">FORECast</a></span><span class="c13 c9">&nbsp;</span></h3><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 407.40px;"><img alt="" src="images/image14.png" style="width: 624.00px; height: 849.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c7"><span class="c1">Add target DNA sequence:<br>AATGTCCCCCAATGGGAAGTTCATCTGGCACTGCCCACAGGTGAGGAGGTCATGATCCCCTTCTGGAGCTCCCAACGGGCCGTGGTCTGGTTCATCATCTGTAAGAATGGCTTCAAGAGGCTCGGCTGTGGTT</span></p><p class="c7"><span class="c1">sgRNA: CTGGAGCTCCCAACGGGCCG</span></p><p class="c7"><span class="c1">Index of PAM: 82 </span></p><p class="c7 c2"><span class="c1"></span></p><h3 class="c7 c21" id="h.lpa4d9q997v5"><span class="c15"><a class="c3" href="https://www.google.com/url?q=https://indelphi.giffordlab.mit.edu/single&amp;sa=D&amp;source=editors&amp;ust=1698959533881369&amp;usg=AOvVaw3Ww0YJYGNtpW-M-HJMkm9p">indelphi</a></span></h3><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 320.00px;"><img alt="" src="images/image7.png" style="width: 624.00px; height: 320.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 46.67px;"><img alt="" src="images/image17.png" style="width: 624.00px; height: 46.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span class="c5">Cut site = 3bp before from PAM sequence (Cas9 protein cleavage site)</span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span class="c5">Left panel: AATGTCCCCCAATGGGAAGTTCATCTGGCACTGCCCACAGGTGAGGAGGTCATGATCCCCTTCTGGAGCTCCCAACGGG<br></span></p><p class="c0"><span class="c5">Right panel: <br>CCGTGGTCTGGTTCATCATCTGTAAGAATGGCTTCAAGAGGCTCGGCTGTGGTT</span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span class="c9 c19">&gt;Results</span></p><p class="c0 c2"><span class="c19 c9"></span></p><p class="c0"><span class="c1">Visualize the prediction of repairs<br>In inDelphi, you additionally select the cell types as well. </span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 245.33px;"><img alt="" src="images/image1.png" style="width: 624.00px; height: 245.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 484.00px;"><img alt="" src="images/image3.png" style="width: 624.00px; height: 484.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0 c2"><span class="c5"></span></p><hr><p class="c0 c2"><span class="c5"></span></p><p class="c0 c2"><span class="c5"></span></p><h2 class="c17" id="h.pq5b31abdv61"><span class="c12">Outcome analysis</span></h2><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 297.33px;"><img alt="" src="images/image11.png" style="width: 624.00px; height: 297.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span class="c9 c20">Background information:</span><span class="c9"><br><br></span><span class="c18 c23">If we transfect Cas9 protein with sgRNA into the cells, each cell will uptake the RNP complex (Cas9+sgRNA) and this will induce the double strand break, and then the cell will repair that break by NHEJ process. Meaning that each cell will have a different sequence in the target region at the end of the repair process and some of them can repair the same as the original. Therefore, if we amplify the target region, we will get the mixed sequence when Cas9 works well. To identify the mixed sequence population, we will use Sanger sequencing or deep sequencing (NGS).</span></p><p class="c0 c2"><span class="c18 c23"></span></p><h3 class="c21 c29" id="h.ec4uh1n8zo5r"><span class="c9 c14">Sanger sequencing</span></h3><p class="c0 c2"><span class="c6"></span></p><p class="c0"><span class="c18 c23">Material</span></p><p class="c0 c10"><span class="c9">Example File 1)<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Control sample: </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://drive.google.com/file/d/1AiI049vMI0uc3H1fm2hXWoRiPg7FkOpT/view?usp%3Dsharing&amp;sa=D&amp;source=editors&amp;ust=1698959533884300&amp;usg=AOvVaw1x0tGqBg4sfls90rAUn4I2">Sanger_ex1_con.ab1</a></span><span class="c9"><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Experiments sample: </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://drive.google.com/file/d/1g8B7bpDUJQYXbgbGMN_7fWZy5HGoJz5M/view?usp%3Dsharing&amp;sa=D&amp;source=editors&amp;ust=1698959533884794&amp;usg=AOvVaw3V1uVzg7myenlSTvxDlPTw">Sanger_ex1_CRISPR.ab1</a></span><span class="c1"><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sgRNA: TGTATGAGTCGAAGATCTCC</span></p><p class="c0 c10 c2"><span class="c1"></span></p><p class="c0 c2 c10"><span class="c1"></span></p><p class="c7 c10"><span class="c9">Example File 2)<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Control sample: </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://drive.google.com/file/d/1nBu-F8fP3gsUXVvF-zOWDf9rzPVZDLKI/view?usp%3Dsharing&amp;sa=D&amp;source=editors&amp;ust=1698959533885369&amp;usg=AOvVaw0XmzHPRC4GgcQaMAfeBJlE">Sanger_ex2_con.ab1</a></span><span class="c9"><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Experiments sample: </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://drive.google.com/file/d/1TW5GlA6xGBE6Ag3gU2E98WdA-PgLAyRj/view?usp%3Dsharing&amp;sa=D&amp;source=editors&amp;ust=1698959533885679&amp;usg=AOvVaw3BNwNY8pLx-VXCl5lVoyvp">Sanger_ex2_CRISPR.ab1</a></span><span class="c1"><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;sgRNA: GGAAGTATACACGCTATTGT</span></p><p class="c0"><span class="c15"><a class="c3" href="https://www.google.com/url?q=https://tide.nki.nl/&amp;sa=D&amp;source=editors&amp;ust=1698959533885944&amp;usg=AOvVaw3iswQ6yLPOk7JE5y3aID8_">TIDE</a></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 296.00px;"><img alt="" src="images/image9.png" style="width: 624.00px; height: 296.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span class="c15"><a class="c3" href="https://www.google.com/url?q=https://ice.synthego.com/%23/&amp;sa=D&amp;source=editors&amp;ust=1698959533886417&amp;usg=AOvVaw1OtegROFT3Oi_eG53b4n4w">ICE analysis</a></span></p><p class="c7"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 548.60px; height: 446.62px;"><img alt="" src="images/image13.png" style="width: 548.60px; height: 446.62px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c7"><span class="c1">Submit control and experiment file into the correct category. And add provided sgRNA information. </span></p><p class="c7 c2"><span class="c1"></span></p><p class="c7"><span class="c1">&gt;Results </span></p><p class="c7"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 634.20px; height: 356.34px;"><img alt="" src="images/image18.png" style="width: 634.20px; height: 356.34px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c7"><span class="c1">Algorithme will decompose the trace of sequencing Chromatogram and will calculate the indel size range and frequency. </span></p><hr><p class="c2 c7"><span class="c1"></span></p><h3 class="c7 c21" id="h.2wohfqkuy20j"><span class="c14 c9">Deep sequencing</span></h3><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 583.00px; height: 327.94px;"><img alt="" src="images/image10.png" style="width: 583.00px; height: 327.94px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><ol class="c24 lst-kix_tpm0vcvzdlxe-0 start" start="1"><li class="c0 c11 li-bullet-0"><span class="c5">NGS sequencing reads will align to the reference </span></li><li class="c0 c11 li-bullet-0"><span class="c5">Analyze the sequence near the quantification window</span></li><li class="c0 c11 li-bullet-0"><span>Summarize the results of editing</span></li></ol><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Material</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c9">Example 1)<br>Fastq file R1: </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://drive.google.com/file/d/1eAHJV7f7l33wSvzJXjkVehD5cWITRXn4/view?usp%3Dsharing&amp;sa=D&amp;source=editors&amp;ust=1698959533887956&amp;usg=AOvVaw2_jaLThp8h2XIogliLtQCu">deep_ex1_r1.fastq.gz</a></span></p><p class="c0"><span class="c9">Fastq file R2: </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://drive.google.com/file/d/1VAXP7J3V9RoJSXLxHI5I0lhIjxy9OgIe/view?usp%3Dsharing&amp;sa=D&amp;source=editors&amp;ust=1698959533888325&amp;usg=AOvVaw0MBoJ4CjbY6AXq8u9lRj0q">deep_ex1_r2.fastq.gz</a></span><span class="c1"><br></span></p><p class="c0"><span class="c1">Amplicon sequence (reference for alignment ):&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;AATGTCCCCCAATGGGAAGTTCATCTGGCACTGCCCACAGGTGAGGAGGTCATGATCCCCTTCTGGAGCTCCCAACGGGCCGTGGTCTGGTTCATCATCTGTAAGAATGGCTTCAAGAGGCTCGGCTGTGGTT</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">sgRNA: CTGGAGCTCCCAACGGGCCG</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 350.67px;"><img alt="" src="images/image4.png" style="width: 624.00px; height: 350.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0"><span class="c1">Submit the reads files and amplicon, sgRNA information. </span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">&gt;Results</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 350.67px;"><img alt="" src="images/image8.png" style="width: 624.00px; height: 350.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0 c2"><span class="c1"></span></p><hr><p class="c0 c2"><span class="c1"></span></p><h2 class="c17" id="h.d1ajf57c875s"><span class="c18 c22">Quiz</span></h2><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Quiz 1) You are going to KO the following gene in humans, please suggest the top three sgRNA sequences. </span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">gene: </span></p><p class="c0"><span class="c1">CTGACGTGCCTCTCCCTCCCTCCAGGAAGCCTACGTGATGGCCAGCGTGGACAACCCCCACGTGTGCCGCCTGCTGGGCATCTGCCTCACCTCCACCGTGCAGCTCATCACGCAGCTCATGCCCTTCGGCTGCCTCCTGGACTATGTCCGGGAACACAAAGACAATATTGGCTCCCAGTACCTGCTCAACTGGTGTGTGCAGATCGCAAAGGTAATCAGGGAAGGGAGATACGGGGAGGGGAGATAAGG</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Quiz 2) For targeting the previous gene, we selected the following sgRNA: ATAGTCCAGGAGGCAGCCGA. In the repair prediction in indelphi, what were the top two abundant repair cases and &nbsp;its percentage. </span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Quiz 3) This is the amplicon sequencing read results from the cells transfected with Cas9 and sgRNA (ATAGTCCAGGAGGCAGCCGA), </span></p><p class="c0"><span class="c9">Fastq file R1: </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://drive.google.com/file/d/1qqMyEVeCwPbuV1_Y9ZqL8_1Eg_fED6lt/view?usp%3Dsharing&amp;sa=D&amp;source=editors&amp;ust=1698959533890296&amp;usg=AOvVaw2kEmLIzZ7QZ0ALBK5AJ_i7">deep_ex2_r1.fastq.gz</a></span></p><p class="c0"><span class="c9">Fastq file R2: </span><span class="c8"><a class="c3" href="https://www.google.com/url?q=https://drive.google.com/file/d/1YY95qUC2ayTrjqVLx7oHD6badBz8jxoi/view?usp%3Dsharing&amp;sa=D&amp;source=editors&amp;ust=1698959533890638&amp;usg=AOvVaw2k6IPiw92BNS40TMqDnnJC">deep_ex2_r2.fastq.gz</a></span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Amplified sequence:</span></p><p class="c0"><span class="c1">TGACGTGCCTCTCCCTCCCTCCAGGAAGCCTACGTGATGGCCAGCGTGGACAACCCCCACGTGTGCCGCCTGCTGGGCATCTGCCTCACCTCCACCGTGCAGCTCATCACGCAGCTCATGCCCTTCGGCTGCCTCCTGGACTATGTCCGGGAACACAAAGACAATATTGGCTCCCAGTACCTGCTCAACTGGTGTGTGCAGATCGCAAAGGTAATCAGGGAAGGGAGATACGGGGAGGGGAGATAA</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">What about the editing efficiency of this sgRNA? </span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0 c2"><span class="c1"></span></p><hr><p class="c0 c2"><span class="c1"></span></p><p class="c0 c2"><span class="c1"></span></p><h2 class="c17" id="h.4mg14oh9jw2e"><span class="c18 c22">Appendix</span></h2><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span class="c5">How to check the Ensembl ID or Refseq number?</span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span class="c5">In Ensembl</span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 314.67px;"><img alt="" src="images/image2.png" style="width: 624.00px; height: 314.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0"><span class="c5">In NCBI<br></span></p><p class="c0"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 305.33px;"><img alt="" src="images/image12.png" style="width: 624.00px; height: 305.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c0 c2"><span class="c5"></span></p><p class="c0 c2"><span class="c5"></span></p><h2 class="c16" id="h.5g3yvv7u6yg"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 426.67px;"><img alt="" src="images/image5.png" style="width: 624.00px; height: 426.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></h2><p class="c0 c2"><span class="c5"></span></p><p class="c0 c2"><span class="c1"></span></p></body></html>